### ISSUE TITLE
Force tika to use tesseract-ocr

### ISSUE THREAD

**Issue by pgrandinetti** [chrismattmann/tika-python#159]
Is there a way to force python-tika to use tesseract?
It cannot parse a scanned PDF, while it works fine if I manually take a screenshot (.png) of a page of the same pdf and give it to tika; the results is correct (I do have tesseract-ocr correctly installed). So I am guessing it does not use tesseract-ocr with the first file because it detects it as PDF.

**Comment by pgrandinetti** [comment]
As a workaround to this, I am first running a subprocess that fires up the `convert` utility (from imagemagick) so that the scanned pdf is converted to an image, and then I run your `parser.from_file` passing the just-created image.
There are several pitfalls in this:
- the subprocess has to be completed before to continue (quite easy with `Popen.wait()`)
- the image file *must* be written to disk *before* tika is called; I am doing this with `os.fsync()`
- what density should be used to convert from pdf to image (this is numerical parameter of the `convert` command)

Even with those two tricks, I find the overall process very unstable. It would awesome if python-tika could support it out of the box

**Comment by chrismattmann** [comment]
this is in the Tika-Server end. If it's configured to use Tesseract (aka if Tesseract is on the path) then it should call it. You could provide a custom config to Tika-Server, and then use tika-python to build that custom config to map the TesseractOCR parser to the specific MIME types you want. If you do that and have a Python programmatic way would love a new PR for it. Thanks.
